system-tests/projects/vueclivue2-configured/package.json

```diff
@@ -10,8 +10,7 @@
     "vue": "^2.6.11"
   },
   "devDependencies": {
-    "@cypress/vue": "^2.0.0",
-    "@cypress/webpack-dev-server": "^1.0.0",
+    "@cypress/webpack-dev-server": "file:../../../npm/webpack-dev-server",
```

Comment:
Not `-fresh` for this and those below, like `system-tests/projects/vuecli5vue3-configured/package.json` above?

Reply:
Those didn't exist at the time and we're going to convert everything to object API anyway in a half-second.